.github/workflows/test_demo.yaml

```diff
@@ -0,0 +1,28 @@
+name: Run Demo
+
+on:
+  pull_request:
+
+jobs:
+  demo:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - name: Check out code
+        uses : actions/checkout@v4
+
+      - name: Run compose demo
+        run: |
+          cp config/config.yaml.example config/config.yaml
+          cp config/iss.json.example config/iss.json
+          cp .env.example .env
+          docker compose -f docker-compose-demo.yml up -d
+          until [ "$(docker inspect data_loader  --format='{{.State.ExitCode}}')" -eq 0 ]; do
+            sleep 5
+          done
+          token=$(curl -s -k https://localhost:8080/tokens | jq -r '.[0]')
+          if [ curl -s -H "Authorization: Bearer $token" http://localhost:8443/metadata/datasets | jq -r .[] != "EGAD74900000101" ]; then
+            echo "demo failed"
+            exit 1
+          fi
```

Comment:
This is not enough since the demo currently seems to fail:
```
mapper       | {"level":"debug","msg":"received a message: {\"type\":\"mapping\",\"dataset_id\":\"EGAD74900000101\",\"accession_ids\":[\"EGAF74900000001\",\"EGAF74900000002\",\"EGAF74900000003\",\"EGAF74900000004\"]}","time":"2023-12-14T08:13:37Z"}
mapper       | {"level":"debug","msg":"Mapping type operation, mapping files to dataset","time":"2023-12-14T08:13:37Z"}
data_loader  | {"routed":true}{"routed":true}{"routed":true}{"routed":true}{"routed":true}
mapper       | {"level":"error","msg":"something went wrong with the DB query: sql: no rows in result set","time":"2023-12-14T08:13:37Z"}
data_loader exited with code 0
mapper       | {"level":"error","msg":"something went wrong with the DB query: sql: no rows in result set","time":"2023-12-14T08:13:39Z"}
mapper       | {"level":"error","msg":"something went wrong with the DB query: sql: no rows in result set","time":"2023-12-14T08:13:43Z"}
mapper       | {"level":"error","msg":"something went wrong with the DB query: sql: no rows in result set","time":"2023-12-14T08:13:51Z"}
mapper       | {"level":"error","msg":"something went wrong with the DB query: sql: no rows in result set","time":"2023-12-14T08:14:07Z"}
```

Better check for files in dataset.